fractor-xml/src/Helper/FlexFormHelperTrait.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorXml\Helper;
+
+trait FlexFormHelperTrait
```

Comment:
AFAICS none of the methods in here relies on any state in the class, so this could become a separate class. Maybe a facade around \DOMElement (that would depend on how exactly it is used, I did not check that further)?

Reply:
I just copied over the class from TYPO3 Rector. Please let me know how this could look like